emails/2015/feb-meetup-call-for-speakers.md

```diff
@@ -0,0 +1,92 @@
+# Title: February meetup, call for speakers, call for video help
+
+## Welcome to 2015
+
+Hey hey, Django Boston! Welcome to 2015! We've got a meetup scheduled for this month, and are planning the talks and events for the rest of the year. Come join us as we learn together!
+
+## Thanks to John Roth and Max Nova
+
+John Baldwin and I (Brian Dant) are stepping in as co-organizers for Max Nova and John Roth. Thanks John and Max for holding it together over the last 18 months or so! (If you have benefited from the contributions from Max and John, please send them a quick note thanking them: uhlenhuthm@gmail.com jroth.solar@gmail.com
+
+## Upcoming Meetup: Feb 26th at Akamai
+
+This month, Shankar Ambady from MassChallenge will be giving a talk on Natural Language Processing using [NLTK](http://www.nltk.org/) and Django. This should be awesome! Shankar has given a talk to Boston Python on NLTK, and now he's adding Django to the mix. RSVP [here](http://www.meetup.com/djangoboston/events/220373782/).
+
+We're still looking for a drink sponsor for the February meetup. If your company is interested in sponsoring, please let us know by responding to this email. Sponsorship will allow us all to go out for drinks together after the meetup, and gives you an opportunity to spread the word about your company and development team.
```

Comment:
@briandant we have a sponsorship page which explains more about how it works: http://www.meetup.com/djangoboston/pages/Sponsoring_a_Django_Boston_Meetup/

So we might want to link to that, which could cut down on the # of questions we receive.